demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -1968,12 +1969,60 @@ def validate_job(self, structure_validator, pack_error_ignore_list):
 
         return is_valid
 
+    def get_all_files_edited_in_pack_ignore(self, modified_files):
+        all_files = self.git_util.get_all_files()
+        all_files_edited_in_pack_ignore = set()
+        for file_path in modified_files:
+            # handle renamed files
+            if isinstance(file_path, tuple):
+                old_file_path = file_path[0]
+                file_path = file_path[1]
+
+            else:
+                old_file_path = file_path
+            if not file_path.endswith(".pack-ignore"):
+                continue
+            old_file_content = get_remote_file(old_file_path, tag="master")
+            config = ConfigParser(allow_no_value=True)
+            config.read_string(old_file_content)
+            old_file_content = self.get_error_ignore_list(config=config)
+            pack_name = get_pack_name(str(file_path))
+            file_content = self.get_error_ignore_list(pack_name)
+            files_to_test = set()
+            for key, value in old_file_content.items():
```

Comment:
we're adding these to the same list and the loops are the same, we can do 
for key, value in old_file_content.items() + file_content.items()

Reply:
But we want to ensure that when we run on old we compare against new and the other way around.

Reply:
ah ok missed that